au/code/au/wrapper_operations_test.cc

```diff
@@ -144,18 +145,22 @@ TEST(CanScaleByMagnitude, MakesScaledWrapperWhenDividingByMagnitude) {
     StaticAssertTypeEq<decltype(mol / PI), UnitWrapper<decltype(Moles{} / PI)>>();
 }
 
-TEST(ForbidsComposingWith, FailsToCompileWhenMultiplyingOrDividingWithForbiddenWrapper) {
-    // Uncomment each line below individually to verify.
+TEST(SupportsRationalPowers, RaisesUnitToGivenPower) {
+    constexpr auto mol = UnitWrapper<Moles>{};
+    StaticAssertTypeEq<decltype(pow<3>(mol)), UnitWrapper<decltype(pow<3>(Moles{}))>>();
+}
+
+TEST(SupportsRationalPowers, EnablesTakingRoots) {
+    constexpr auto mol = UnitWrapper<Moles>{};
+    StaticAssertTypeEq<decltype(root<8>(mol)), UnitWrapper<decltype(root<8>(Moles{}))>>();
+}
 
-    // UnitWrapper<Meters>{} * meters_pt;
```

Comment:
Q: Are these tests irrelevant now?

Reply:
Oops... actually, no!  We deleted `ForbidsComposingWith` in #227, so I thought we forgot to remove this, but reading that PR summary I see that leaving these here was intentional.